Moya.podspec

```diff
@@ -31,7 +31,7 @@ Pod::Spec.new do |s|
   s.subspec "ReactiveCocoa" do |ss|
     ss.source_files = "Moya/ReactiveCocoa/*.swift"
     ss.dependency "Moya/ReactiveCore"
-    ss.dependency "ReactiveCocoa", "~> 4.0-alpha.1"
+    ss.dependency "ReactiveCocoa", "= 4.0.0-alpha-3"
```

Comment:
I don't think `~> 4.0.0-alpha.3` will do what you want, I think it will just equals. Yeah, probably have to do the exact version, given that they've broke API compat each time, seems logical to presume that in the future.